test/bindings_workflows_test.go

```diff
@@ -0,0 +1,140 @@
+// The MIT License
```

Comment:
This file doesn't have any tests. It is more like `workflow_definition_factory.go`.

Reply:
removing _test prefix makes it public :(